lib/src/obsoleted/metrics_analyzer.dart

```diff
@@ -285,8 +290,17 @@ class MetricsAnalyzer {
   Iterable<Issue> _checkOnCodeIssues(
     Suppression ignores,
     InternalResolvedUnitResult source,
+    String filePath,
```

Comment:
@incendial I'd suggest to receive here absolute path

Reply:
Discussed it, decided to keep as is for now.